pytube/contrib/search.py

```diff
@@ -39,10 +39,9 @@ def completion_suggestions(self):
         :returns:
             A list of autocomplete suggestions provided by YouTube for the query.
         """
-        if self._completion_suggestions:
-            return self._completion_suggestions
```

Comment:
This does not seem to be a redundant statemtent. It avoids updating the cache if the value has been set once.

Reply:
oh